src/apple/deps/update.rs

```diff
@@ -37,34 +51,88 @@ impl Formula {
     }
 }
 
+fn parse_gem_outdated_string(revision: &str) -> Result<Formula, RegexError> {
+    let caps = regex!(r"(?P<name>.+) \((?P<installed_version>.+) < (?P<latest_version>.+)\)")
+        .captures(revision)
+        .ok_or_else(|| RegexError::SearchFailed {
+            revision: revision.to_owned(),
+        })?;
+
+    let name = util::get_string_for_group(&caps, "name", revision)
+        .map_err(RegexError::InvalidCaptureGroup)?;
+    let installed_version = util::get_string_for_group(&caps, "installed_version", revision)
+        .map_err(RegexError::InvalidCaptureGroup)?;
+    let current_version = util::get_string_for_group(&caps, "current_version", revision)
+        .map_err(RegexError::InvalidCaptureGroup)?;
+
+    Ok(Formula {
+        name,
+        installed_versions: vec![installed_version],
+        current_version,
+    })
+}
+
+#[derive(Deserialize)]
+struct Raw {
+    formulae: Vec<Formula>,
+}
+
 #[derive(Debug)]
 pub struct Outdated {
     packages: Vec<Formula>,
 }
 
 impl Outdated {
-    pub fn load() -> Result<Self, OutdatedError> {
-        #[derive(Deserialize)]
-        struct Raw {
-            formulae: Vec<Formula>,
-        }
+    fn outdated_gem_command() -> Result<String, OutdatedError> {
```

Comment:
This should be `outdated_gem_deps` and match the level of abstraction `outdated_brew_deps` provides